frontend/src/scenes/data-warehouse/new/NewSourceWizard.tsx

```diff
@@ -153,7 +153,7 @@ function FirstStep({ disableConnectedSources }: Pick<NewSourcesWizardProps, 'dis
 
             <p>
                 Data will be synced to PostHog and regularly refreshed.{' '}
-                <Link to="https://posthog.com/docs/data-warehouse/setup#stripe">Learn more</Link>
+                <Link to="https://posthog.com/docs/cdp/sources">Learn more</Link>
```

Comment:
It looks like we were going to stripe here and the equivalent link I think would be `https://posthog.com/docs/cdp/sources/stripe`
Just making sure that this is indeed the change we wish to make.

Reply:
Let's go with what you have, I think it makes a lot more sense!